tests/blackbox.py

```diff
@@ -408,41 +438,47 @@ def dryrun_on_sequence(
         inputs: list[Sequence[PyTypes]],
         compiler_version=6,
     ) -> list[DryRunInspector]:
+        teal = self.compile(compiler_version)
         return _MatchMode(
+            self,
             app_case=lambda: DryRunExecutor.dryrun_app_on_sequence(
                 algod=algod_with_assertion(),
-                teal=self.compile(compiler_version),
+                teal=teal,
                 inputs=inputs,
-                abi_argument_types=self.abi_argument_types(),
-                abi_return_type=self.abi_return_type(),
+                abi_method_signature=self.abi_method_signature(),
+                omit_method_selector=True,
```

Comment:
The parameters `abit_argument_types` and `abi_return_type` are removed from graviton's `DryRunExecutor` in favor of `abi_method_signature`